charts/vela-core/templates/kubevela-controller.yaml

```diff
@@ -142,16 +142,16 @@ spec:
             initialDelaySeconds: 90
             periodSeconds: 5
           volumeMounts:
-            - mountPath: {{ .Values.certificate.mountPath }}
```

Comment:
why remove the mountPath helm value and use a hardcode one

Reply:
I didn't see any case, to set the mount path manually, but I can re-add it again.

Reply:
thanks, please add it back,